chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -268,7 +268,7 @@ sync
       <package>biosdevname</package>
       <package>netcat</package>
       <package>ruby2.1-rubygem-chef</package>
-<% if @is_ses -%>
+<% if @is_ses and @target_platform_version == "12.0" -%>
```

Comment:
I must be missing something. What's the underlying problem? In this context `@is_ses` should only be true if the cloud repos are missing but storage repos are available.